src/tools/rbd_nbd/rbd-nbd.cc

```diff
@@ -354,7 +354,6 @@ class NBDServer
     void* entry() override
     {
       (server.*func)();
-      server.shutdown();
```

Comment:
I test it: map a nbd device, and use fio to write, and unmap. I didn't see the issue. 

I thought this shutdown also called in the function "stop()". Please correct me.